src/InstrumentationEngine.Attach/InstrumentationEngine.Attach.csproj

```diff
@@ -13,10 +15,39 @@
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
     <Nullable>Enable</Nullable>
   </PropertyGroup>
+  <ItemGroup>
+    <XsdSchemaFile Include="InstrumentationConfigurationSources.xsd">
+      <Namespace>Microsoft.InstrumentationEngine</Namespace>
+      <SubType>Designer</SubType>
+      <Visible>False</Visible>
+    </XsdSchemaFile>
+  </ItemGroup>
+  <ItemGroup>
+    <EmbeddedResource Include="InstrumentationConfigurationSources.xsd" />
+  </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.Diagnostics.NETCore.Client" Version="0.2.61701" />
     <PackageReference Include="Microsoft.VisualStudioEng.MicroBuild.Core" Version="$(MicroBuildCoreVersion)" />
   </ItemGroup>
   <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" />
   <Import Project="$(EnlistmentRoot)\build\Managed.targets" />
+  <Target Name="PrepareXsdGeneration" Condition="'@(XsdSchemaFile)' != ''">
+    <PropertyGroup>
+      <FullIntermediateOutputPath>$([System.IO.Path]::GetFullPath('$(IntermediateOutputPath)'))</FullIntermediateOutputPath>
+    </PropertyGroup>
+    <ItemGroup>
+      <XsdSchemaFile>
+        <OutputFile>$(FullIntermediateOutputPath)%(FileName).cs</OutputFile>
+        <Language>CS</Language>
+      </XsdSchemaFile>
+    </ItemGroup>
+    <ItemGroup>
+      <Compile Condition="'%(XsdSchemaFile.OutputFile)' != ''" Include="%(XsdSchemaFile.OutputFile)" />
+    </ItemGroup>
+    <Error Condition="'%(XsdSchemaFile.Language)' == ''" Text="Project type is expected to by one of 'CSharp' or 'VisualBasic' for file %(XsdSchemaFile.FullPath)" />
+  </Target>
```

Comment:
All of this can be added to the XsdSchemaFile element above, especially since the generated cs file is committed:
- Add OutputFile and Language elements to the XsdSchemeFile; change the value of OutputFile to point at the cs file that is committed rather than something under the intermediate directory.
- Compile element is not needed since it will be automatically picked up by the automatic inclusion of code files.